Gemfile.lock

```diff
@@ -0,0 +1,251 @@
+PATH
```

Comment:
(the appraisal lockfiles probably shouldn't be checked in either ... i'll check how easy they'd be to remove)